dev/bots/test.dart

```diff
@@ -204,11 +205,19 @@ Future<Null> _runCoverage() async {
 Future<Null> _pubRunTest(
   String workingDirectory, {
   String testPath,
-}) {
+  int retries = 1,
```

Comment:
Maybe name this "maxAttempts" since `retries==1` implies we'll retry if the first attempt fails.